build.sh

```diff
@@ -258,11 +258,9 @@ echo "Current directory: $(pwd)"
 echo "Build directory: ${build_dir}"
 echo "Install directory: ${install_dir}"
 
-if [ "$n_parallel"  -gt "10" ]; then
-    jobs=$(( (n_parallel / 10) * 9 ))
-else
-    jobs=16
-fi
+
+ jobs=$(n_parallel - 1)
```

Comment:
1. Indentation.
2. Should probably be: jobs=$(( n_parallel - 1 ))

Are you sure? This could cause the machine to slow to a crawl in certain cases...